Packs/Luminate/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Luminate",
     "description": "Enrich your report and Respond to incidents with Luminate",
-    "support": "partner",
-    "currentVersion": "1.0.0",
+    "support": "community",
+    "currentVersion": "1.0.1",
     "author": "Luminate",
     "url": "https://www.paloaltonetworks.com/cortex",
```

Comment:
If this goes to the community we need to remove it, if we change to XSOAR then keep it.

Reply:
Remove URL

Reply:
done